biolink/api/graph/endpoints/node.py

```diff
@@ -1,44 +1,74 @@
 import logging
 
 from flask import request
-from flask_restplus import Resource
-from biolink.datamodel.serializers import association, bbop_graph
+from flask_restplus import Resource, inputs
+from biolink.datamodel.serializers import association, bbop_graph, bio_object
+from biolink.error_handlers import NoResultFoundException
 from scigraph.scigraph_util import SciGraph
+from scigraph.model.BBOPGraph import BBOPGraph
 from biolink.api.restplus import api
 from biolink.settings import get_biolink_config
 
 log = logging.getLogger(__name__)
 
-parser = api.parser()
 sg = SciGraph(get_biolink_config()['scigraph_data']['url'])
 
 @api.doc(params={'id': 'CURIE e.g. HP:0000465'})
 class NodeResource(Resource):
 
-    @api.expect(parser)
-    @api.marshal_list_with(bbop_graph)
+    @api.marshal_list_with(bio_object)
     def get(self, id):
         """
         Returns a graph node.
 
         A node is an abstract representation of some kind of entity. The entity may be a physical thing such as a patient,
         a molecular entity such as a gene or protein, or a conceptual entity such as a class from an ontology.
         """
-        args = parser.parse_args()
-        
-        return sg.graph(id)
+        g = sg.bioobject(id)
```

Comment:
I favor descriptive variable names (excepting i,j,k,v), any scoping issues using graph?

Reply:
No issues with adding descriptive variable names. I'll use `graph` instead of `g`.